src/content/en/updates/posts/2016/03/web-animations-improvements.markdown

```diff
@@ -0,0 +1,69 @@
+---
+layout: updates/post
+title: "Web Animations Improvements in Chrome 50"
+description: "Better specification compliance and new features coming in Chrome 50."
+published_on: 2016-03-15
+updated_on: 2016-03-15
+authors:
+  - adanilo
+tags:
+  - animation
+  - chrome50
+---
+
+The [Web Animations API](http://w3c.github.io/web-animations/), which first shipped in Chrome 36, provides convenient Javascript control of
+animations in the browser, and is also being implemented in Gecko and WebKit.
+
+Chrome 50 introduces changes to improve interoperability with other browsers
+and to be more compliant with the spec. These changes are:
+
+  * Cancel events
+  * Animation.id
+  * State change for the pause() method
+  * Deprecation of dashed names as keys in keyframes
+
+### Cancel Events
+
+The [Animation](http://w3c.github.io/web-animations/#the-animation-interface) interface includes a method to cancel an animation, funnily enough called [cancel()](http://w3c.github.io/web-animations/#dom-animation-cancel). Chrome 50 implements firing of the cancel event when the method is called as
+per the spec, which triggers event handling through the [‘oncancel’](http://w3c.github.io/web-animations/#dom-animation-oncancel) attribute if it’s been initialized.
+
+### Support for Animation.id
+
+When you create an animation using [element.animate()](http://w3c.github.io/web-animations/#dom-animatable-animate) you can pass in a number of properties. For example, here’s an example of
+animating opacity on an object:
+
+
+```
```

Comment:
wrap major code like this in `{% highlight javascript %}` and `{% endhighlight %}`